GVRf/Framework/framework/src/main/java/org/gearvrf/io/GearCursorController.java

```diff
@@ -520,7 +516,10 @@ public void run()
                 for (MotionEvent e : mMotionEvents)
                 {
                     activity.dispatchTouchEvent(MotionEvent.obtain(e));
+                    e.recycle();
```

Comment:
This recycle is causing problems with gvr-events and I doubt only with it.
```
01-23 08:26:39.176 12053 12053 E AndroidRuntime: java.lang.RuntimeException: MotionEvent { action=ACTION_MOVE, actionButton=0, id[0]=0, x[0]=187.0, y[0]=67.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=BUTTON_PRIMARY, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=176380, downTime=176346, deviceId=0, source=0x100008 } recycled twice!
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at android.view.InputEvent.recycle(InputEvent.java:120)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at android.view.MotionEvent.recycle(MotionEvent.java:2053)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at org.gearvrf.io.GearCursorController$EventHandlerThread$SendEvents.run(GearCursorController.java:519)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:751)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6776)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1520)
01-23 08:26:39.176 12053 12053 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1410)
```

Probably removing it is enough. That is what I did locally.

Reply:
I guess the duped motion event needs to be recycled instead.